auditorium/src/views/components/_shared/account-picker.js

```diff
@@ -47,9 +47,11 @@ const AccountPicker = (props) => {
         )
       })
     body = (
-      <ul class='flex flex-wrap list pl0 mt0 mb3 grow-list b--moon-gray'>
```

Comment:
I think this means we can now remove the `grow-list.css` filetoo?

Reply:
nope its still needed for the "Show data from the last" tab